knimin/tests/test_pm_plate_list.py

```diff
@@ -0,0 +1,56 @@
+from unittest import main
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class TestPMPlateListHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get(
+            '/pm_plate_list/')
+        self.assertEqual(response.code, 200)
+        self.assertTrue(
+            response.effective_url.endswith('?next=%2Fpm_plate_list%2F'))
+
+    def test_get(self):
+        self.mock_login_admin()
+        response = self.get('/pm_plate_list/')
+        self.assertEqual(response.code, 200)
+        # Test the page title
+        self.assertIn('Sample Plate List', response.body)
+        # Test the first sample plate
+        p1 = db.get_sample_plate_list()[0]
```

Comment:
Looking at this test, it sounds like this test is random.  It depends on the state of the database.

Is it possible to hard code p1 to make it less random?  So that we don't have to worry about the ordering in the sample_plate_list()